src/app/components/MediaLoader/types.ts

```diff
@@ -63,6 +63,10 @@ export type PlaylistItem = {
   serviceID?: string;
 };
 
+export type LegacyPlayListItem = {
+  href: string;
```

Comment:
Does this also need to include `kind`: https://github.com/bbc/morph-modules/blob/0f16bd34fad7339f0a61e5dcff3e489dbba9de3d/news-av-player/src/legacy-media/index.js#L28